tsconfig.json

```diff
@@ -7,6 +7,7 @@
     "allowJs": true,
     "alwaysStrict": true,
     "checkJs": false,
+    "declaration": true,
```

Comment:
My editor was reporting an error in the file, so this is an unrelated fix for that